src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseController.java

```diff
@@ -50,7 +51,7 @@ public class ManageCaseController {
     public ResponseEntity<CaseDetails> getUserCaseDetails(
         @RequestHeader(AUTHORIZATION) String authorization,
         @RequestBody CaseRequest caseRequest) {
-        var caseDetails = caseService.getUserCase(authorization, caseRequest);
+        var caseDetails = caseService.getUserCase(authorization, caseRequest.getCaseId());
```

Comment:
In Callbacks I had a ticket to change all vars to actual Types.

Reply:
resolved